nixos/modules/services/misc/ringboard.nix

```diff
@@ -0,0 +1,64 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.ringboard;
+in
+{
+
+  options.services.ringboard = {
+    enable = lib.mkEnableOption "Ringboard clipboard manager";
+
+    package = lib.mkPackageOption pkgs "ringboard" { };
+  };
+
+  config = lib.mkIf cfg.enable {
+    systemd.user.services.ringboard-server = {
+      description = "Ringboard server";
+      documentation = [ "https://github.com/SUPERCILEX/clipboard-history" ];
+      after = [ "multi-user.target" ];
+      wantedBy = [ "multi-user.target" ];
+      serviceConfig = {
+        Type = "notify";
+        Environment = "RUST_LOG=trace";
```

Comment:
```suggestion
        environment.RUST_LOG = "trace";
```

Reply:
Needs to be placed outside of `serviceConfig` if applied

Reply:
Needs to be placed outside of `serviceConfig` if applied

Reply:
@magnetophon still relevant I believe